packages/123done/.eslintrc.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Interesting. Not sure why the file was deleted+added. I thought <kbd>git mv .eslintrc .eslintrc.json</kbd> was the correct command. 🤷